hieradata/class/vagrant/mongo.yaml

```diff
@@ -0,0 +1,47 @@
+---
+mongodb::backup::s3_backups: true
+
+mongodb::s3backup::backup::aws_access_key_id: '1234567890abcd' 
+mongodb::s3backup::backup::aws_secret_access_key: 'ABCDEFGHIJKLMNOP1234567890'
+mongodb::s3backup::backup::s3_bucket: govuk-mongodb-backup-s3-vagrant
+mongodb::s3backup::backup::standalone: true
```

Comment:
Yep, so I think it can be removed as part of 56b6088dc268e5cb7a280a4d554eab02c0979d7a to keep the hieradata tidy.